### ISSUE TITLE
Tech debt: Remove all MapVeu/MapVEu application code

### ISSUE THREAD

**Issue by bobular** [VEuPathDB/web-eda#1300]
Before we start on the integrated full screen map/viz work, we should clear the decks by getting rid of the placeholder MapVEu application code.

**Comment by bobular** [comment]
Not needed now we are reviving the standalone mapveu app
